src/main/java/thePackmaster/powers/rimworldpack/MoodPower.java

```diff
@@ -0,0 +1,190 @@
+    }
+
+
+
+    public void getMentalBreak(){
+        BreaksAndInspirations.mentalBreak(owner);
+    }
+
+    @Override
+    public void updateDescription() {
+        if(amount > 0 )
+            description = DESCRIPTIONS[0] + amount + DESCRIPTIONS[1];
+        else
+            description = DESCRIPTIONS[0]+ amount + DESCRIPTIONS[2];
+    }
+
+
+    public static class BreaksAndInspirations{
+
+        public static void inspiration(AbstractCreature owner){
+            int rand = AbstractDungeon.cardRandomRng.random(2);
+            for(int i = 0; i < 20; ++i)
+                AbstractDungeon.effectsQueue.add(new ShineSparkleEffect(owner.hb.x + (float)Math.random() * owner.hb.width, owner.hb.y + (float)Math.random() * owner.hb.height));
+            Wiz.atb(new TalkAction(true, inspirationStrings.TEXT[rand], 2.5f, 2.5f));
+            switch (rand)
+            {
+                case 1:
+                    Wiz.applyToSelf(new BufferPower(owner, 1));
+                    break;
+                case 2:
+                    Wiz.atb(new DrawCardAction(2));
+                    Wiz.atb(new GainEnergyAction(2));
+                    break;
+                default:
+                    fast(owner);
+            }
+        }
+
+        private static void fast(AbstractCreature owner){
+
+            int rand = new Random().nextInt(10);
+            if(rand == 0)
+                Wiz.atb(new SFXAction(modID + "fast"));
+
+            Wiz.applyToSelf(new DupePower(owner, 1));
+
+            if(rand == 0)
+            {
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new WaitAction(.1f));
+                Wiz.atb(new VFXAction(new WhirlwindEffect(), 0.5f));
+                Wiz.atb(new VFXAction(new WhirlwindEffect(), 0.5f));
+                Wiz.atb(new VFXAction(new WhirlwindEffect(), 0.5f));
+                Wiz.atb(new VFXAction(new WhirlwindEffect(), 0.5f));
+                Wiz.atb(new VFXAction(new WhirlwindEffect(), 0.5f));
+                Wiz.atb(new VFXAction(new WhirlwindEffect(), 0.5f));
+                Wiz.atb(new VFXAction(new WhirlwindEffect(), 0.5f));
+            }
+        }
+
+        public static void mentalBreak(AbstractCreature owner)
+        {
+            int rand = AbstractDungeon.cardRandomRng.random(3) + 4;
+            Wiz.atb(new VFXAction(owner, new VerticalAuraEffect(new Color(1, 0, 0, .5f), owner.hb.cX, owner.hb.cY), 0.25F));
+            Wiz.atb(new TalkAction(true, inspirationStrings.TEXT[rand], 2.5f, 2.5f));
```

Comment:
Instead of adding 4 here and subtracting 4 below, I'd suggest juts adding 4 here.

Or even better, this would probably be easier to manage if you split the inspirations and mental breaks into two separate entries in UIStrings. That will let you remove the random +- 4 here entirely, which is a good thing -- right now, if you ever add or remove an inspiration, it will also require you to change the number here. Easy to forget. I think it would also be easier to understand for any hypothetical translator (instead of having an entry that says `===Mental Breaks===` to separate the two).

Reply:
Sure thing, I can do that